etc/picongpu/hypnos-hzdr/k20_picongpu.profile.example

```diff
@@ -25,11 +25,11 @@ then
         module load cmake/3.7.2
         module load boost/1.62.0
         module load cuda/8.0
-        module load openmpi/1.8.6.kepler.cuda80
+        module load openmpi/2.1.2.kepler.cuda80
 
         # Plugins (optional)
         module load pngwriter/0.7.0
-        module load hdf5-parallel/1.8.15 libsplash/1.6.0
+        module load hdf5-parallel/1.8.20 libsplash/1.6.0
```

Comment:
QUESTION:
have you tested libspalash/1.6.0 with hdf5-parallel/1.8.20. According to the admins comment, only libsplash/1.7.0 should work.

Reply:
I am testing it right now, let's see

Reply:
works